Source/SelfService/Web/customer/viewAll.tsx

```diff
@@ -57,7 +53,9 @@ export const ViewAll: React.FunctionComponent<Props> = (props) => {
             <ul>
                 {customers.map((customer) => {
                     return <li key={`${customer.id}`}>
-                        <p>{customer.name} ({customer.id})</p>
+                        <a href={`/admin/customer/${customer.id}`}>
```

Comment:
This doesn't work, it might do locally.
When running in dev or production it has the suffix "selfservice".

The current way I have this working is via history api.

```ts
const href = `/admin/customer/${customer.id}`;
history.push(href);
```

I am hopeful once we get @pavsaund into the code base, we will either have this cleared up, or someone will help me understand the insanity of react ;).

Reply:
I care :), its our little garden.
Nice work.